### DISCUSSION TITLE
DONE - Create complementary illustrations for new Earn lessons

### DISCUSSION THREAD

**Discussion by pretyflaco** [GaloyMoney/galoy#2147] [Bounties]
## Description
We're extending the Earn section of the Bitcoin Beach Wallet app - it's a place where users of the app can learn about bitcoin in short lessons and immediately earn sats for completing a quiz.

There are 99 new lessons in 15 chapters in which users can learn more about money and bitcoin every day. 
You can see all lessons [here](https://dev.galoy.io/galoy-earn/), new ones starting at [Chapter 201 - The Origins of Money](https://dev.galoy.io/galoy-earn/chapter-201-the-origins-of-money)

For these new lessons, 99 complementary illustrations are required. 

### Specification
The new illustrations should be a continuation of the original four chapters starring Badger, the friendly honeybadger mascot of Earn that helps illustrate abstract concepts.
![201 01](https://user-images.githubusercontent.com/32958103/211322926-af8073be-463b-4fe0-af15-1c4ee14ae5a2.png)
 
All original illustrations can be found [here](https://github.com/GaloyMoney/galoy-mobile/tree/main/app/screens/earns-screen).

The new illustrations should be delivered in SVG file format.

For discussions, questions and feedback, you can join the [#Education channel on Mattermost](https://chat.galoy.io/galoy/channels/education)

## Task
Deliver 99 complementary illustrations in SVG format for the lessons in Chapter 201 to Chapter 603.

## Definition of Done
Submit a PR that gets merged to [galoy/earn](https://github.com/GaloyMoney/earn) that adds illustrations for all new lessons
If you want to work on this bounty, please leave a comment below!

## Bounty
5M Sats (0.05 BTC)

## Bounty Patrons
[Galoy](https://galoy.io/)

**Comment by Dave95p** [comment]
Hi! Im interested on working on this bounty!

**Comment by artdesignbySF** [comment]
Hi, checking it out now. I will probably have time in a week or two.

**Reply by artdesignbySF** [reply]
So discussed this with @pretyflaco, I'm ready to start planning this in. Hope to start late next week. Soft deadline, end of January. Hard deadline 8th of February.

**Comment by pretyflaco** [comment]
PSA: Bounty has been picked up by @artdesignbySF and is in progress - 1st in priority to be considered

Next in line:
2nd [Oliver](https://twitter.com/Stromens)
3rd @Dave95p

**Reply by artdesignbySF** [reply]
Thank you @pretyflaco!

**Reply by artdesignbySF** [reply]
I will start work sometime next week.

**Reply by artdesignbySF** [reply]
Will make a start today.

**Reply by artdesignbySF** [reply]
Alright. LMK if any adjustments are needed or if you find errors somewhere. This was fun. Thank you for the job! I'm honored and privileged to be able to help out this amazing project!

**Comment by Ahmadkhan02** [comment]
has this been completed? If not then i would love to work on this

**Reply by pretyflaco** [reply]
Yes, this bounty has been completed already. Results can be viewed here: https://dev.galoy.io/galoy-earn/chapter-201-the-origins-of-money/201.01-lesson-origins-of-money
